Submitted By: R. Bosch <remybosch -at- zonnet -dot- nl>
Date: 2004-02-05
Initial Package Version: 1.1.1a
Origin: openoffice-1.1.0-jdk-1.4.2-fix.patch (checked/remade)
Description: Fixes compilation with jdk-1.4.2

diff -ru oo_1.1.1a_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOAddInCreateIterator.java oo_1.1.1a_src-new/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOAddInCreateIterator.java
--- oo_1.1.1a_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOAddInCreateIterator.java	2003-03-26 14:40:24.000000000 +0100
+++ oo_1.1.1a_src-new/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOAddInCreateIterator.java	2004-02-04 17:08:10.000000000 +0100
@@ -200,7 +200,7 @@
         index--;
     }
     
-    public org.openide.WizardDescriptor$Panel current() {
+    public org.openide.WizardDescriptor.Panel current() {
         return panels[index];
     }
     
diff -ru oo_1.1.1a_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOIDLWizardIterator.java oo_1.1.1a_src-new/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOIDLWizardIterator.java
--- oo_1.1.1a_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOIDLWizardIterator.java	2002-09-03 18:25:06.000000000 +0200
+++ oo_1.1.1a_src-new/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOIDLWizardIterator.java	2004-02-04 17:08:10.000000000 +0100
@@ -207,7 +207,7 @@
    index--;
   }
 
-  public org.openide.WizardDescriptor$Panel current()
+  public org.openide.WizardDescriptor.Panel current()
    {
      return panels[index];
    }
diff -ru oo_1.1.1a_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOServiceCreateIterator.java oo_1.1.1a_src-new/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOServiceCreateIterator.java
--- oo_1.1.1a_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOServiceCreateIterator.java	2003-03-26 14:40:24.000000000 +0100
+++ oo_1.1.1a_src-new/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOServiceCreateIterator.java	2004-02-04 17:08:10.000000000 +0100
@@ -214,7 +214,7 @@
         index--;
     }
     
-    public org.openide.WizardDescriptor$Panel current() {
+    public org.openide.WizardDescriptor.Panel current() {
         return panels[index];
     }
     
